hack/build.sh

```diff
@@ -58,19 +58,18 @@ function digest() {
 	ret=$(docker inspect --format='{{index .RepoDigests 0}}' "${IMAGE}")
 }
 
-docker buildx build --push --platform "linux/amd64,linux/ppc64le"  -t "${REGISTRY}/${NAMESPACE}/quay-operator:${TAG}" .
+docker buildx build --push --platform "linux/amd64,linux/ppc64le,linux/s390x"  -t "${REGISTRY}/${NAMESPACE}/quay-operator:${TAG}" .
 digest "${REGISTRY}/${NAMESPACE}/quay-operator:${TAG}" OPERATOR_DIGEST
 
-digest "${REGISTRY}/${NAMESPACE}/quay:${TAG}" QUAY_DIGEST
+digest "${REGISTRY}/${NAMESPACE}/quay:3.8.5" QUAY_DIGEST
 digest "${REGISTRY}/${NAMESPACE}/clair:nightly" CLAIR_DIGEST
 digest "${REGISTRY}/${NAMESPACE}/quay-builder:${TAG}" BUILDER_DIGEST
 digest "${REGISTRY}/${NAMESPACE}/quay-builder-qemu:3.9.0" BUILDER_QEMU_DIGEST
+digest docker.io/redis:7.0 REDIS_DIGEST
 # shellcheck disable=SC2034
-POSTGRES_DIGEST='quay.io/sclorg/postgresql-13-c9s@sha256:593910f2d4b895f4924261a3b8b2aa6457892100a01a0c0ad661cd378d810d65'
+POSTGRES_DIGEST='quay.io/sclorg/postgresql-15-c9s@sha256:593910f2d4b895f4924261a3b8b2aa6457892100a01a0c0ad661cd378d810d65'
```

Comment:
Customers have data on their volume in Postgres 13 format. They'll need to migrate it if you change Postgres version. In case of the operator, this migration should be automated.
It's better to keep Postgres 13.

Reply:
We should be ok with centos/postgresql-12-centos7 being a non-multiarch image, as it's only for upgrades, and we expect upgrades only on x86.